assets/scss/components/stencil/navPages/_navPages.scss

```diff
@@ -209,7 +209,7 @@
         }
     }
 
-    &:hover {
+    &:hover, &.activePage {
```

Comment:
can we just use `active` as a class name so it is uniform across the codebase. We can always refer to it in a nested way for css.

Reply:
you can always provide specific override for nav only using something like `.nav .active`